test/external-modules/die-with-dignity/src/main/java/org/elasticsearch/DieWithDignityPlugin.java

```diff
@@ -25,10 +25,6 @@
 
 public class DieWithDignityPlugin extends Plugin implements ActionPlugin {
 
-    public DieWithDignityPlugin() {
-        assert System.getProperty("die.with.dignity.test") != null : "test should pass the `die.with.dignity.test` property";
```

Comment:
Because this module is now installed in all snapshot builds, this plugin is loaded in all other test suites where this system property would not be configured. This is why this assertion has to be dropped.